lib/node_modules/@stdlib/math/base/special/sincos/README.md

```diff
@@ -0,0 +1,75 @@
+# sincos
+
+> Simultaneously compute the [sine][sine] and [cosine][cosine] of a number.
+
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var sincos = require( '@stdlib/math/base/special/sincos' );
+```
+
+#### sincos( \[out,\] x )
+
+Simultaneously computes the [sine][sine] and [cosine][cosine] of a `number` (in radians).
+
+``` javascript
+var v = sincos( 0.0 );
+// returns [ ~0.0, ~1.0 ]
+
+v = sincos( Math.PI/2.0 );
+// returns [ ~0.0, ~1.0 ]
+
+v = sincos( -Math.PI/6.0 );
+// returns [ ~-0.5, ~0.866 ]
+```
+
+By default, the function returns the sine and cosine as a two-element `array`. To avoid extra memory allocation, the function supports providing an output (destination) object.
+
+``` javascript
+var out = new Float64Array( 2 );
+
+var v = sincos( out, 0.0 );
+// return <Float64Array>[ ~0.0, ~1.0 ]
+
+var bool = ( v === out );
+// returns true
+```
+
+
+</section>
+
+<!-- /.usage -->
+
+
+<section class="examples">
+
+## Examples
+
+``` javascript
+var linspace = require( '@stdlib/math/utils/linspace' );
+var sincos = require( '@stdlib/math/base/special/sincos' );
+
+var x = linspace( 0.0, 2.0*Math.PI, 100 );
+var i;
+
+for ( i = 0; i < x.length; i++ ) {
+    console.log( sincos( x[ i ] ) );
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+
+<section class="links">
+
+[sine]: https://en.wikipedia.org/wiki/Sine
```

Comment:
Can we point to the base special function here?

`[@stdlib/math/base/special/sine]: https://github.com/stdlib-js/stdlib`

Ditto for cosine. This always us to direct people to our internal docs before sending them outbound.